src/cli.js

```diff
@@ -179,13 +184,13 @@ function installCb(err) {
     console.log(`${C.errorText} ${err.message}`);
     process.exit(1);
   }
-  let successMessage = `${C.successText} ${
-    packageName
-  } and its peerDeps were installed successfully.`;
+  let successMessage = `${
+    C.successText
```

Comment:
I think the linter can't make up it's mind regarding where it wants to do those line breaks...